index.html

```diff
@@ -2636,12 +2636,12 @@ <h2 id="docconformance">
             </td>
             <td>
               <p>
-                <code>role=<a href="#index-aria-cell">cell</a></code> if a
-                descendant of a `table` element.
+                <code>role=<a href="#index-aria-cell">cell</a></code> if the ancestor
+                `table` element has `role=table`.
```

Comment:
```suggestion
                `table` element is exposed as a `role=table`.
```

Reply:
want to change the wording here to not give the impression that a `<table>` *needs* a `role=table` to be exposed correctly.